t/t1200-stats-basic.t

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+#
+
+test_description='Test stats collection and sending'
+
+. `dirname $0`/sharness.sh
+
+test_under_flux 2
+
+udp=$SHARNESS_TEST_SRCDIR/scripts/stats-listen.py
+timeout=$SHARNESS_TEST_SRCDIR/scripts/run_timeout.py
+plugin_i=${FLUX_BUILD_DIR}/t/stats/.libs/stats-immediate.so
+plugin_b=${FLUX_BUILD_DIR}/t/stats/.libs/stats-basic.so
+
+test_expect_success 'load jobtap plugin' '
+	flux jobtap load $plugin_i && flux jobtap load $plugin_b &&
+	flux jobtap list > plugins &&
+	grep stats-basic plugins && grep stats-immediate plugins
+'
```

Comment:
One more comment. I don't think we need this test to run under `test_under_flux`, nor have the jobtap plugins loaded here. I may have missed something but the rest of the functionality tests all use `flux start`, so the "outer" instance created by `test_under_flux` actually isn't used, nor are the loaded plugins.

If that is true then you coudl just remove `test_under_flux 2` and the `load jobtap plugin` test.